src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -10,18 +13,21 @@
 public class AuthenticationServiceImpl implements AuthenticationService {
     private final UserService userService;
     private final ShoppingCartService shoppingCartService;
+    private final RoleService roleService;
 
-    public AuthenticationServiceImpl(UserService userService,
+    public AuthenticationServiceImpl(UserService userService, RoleService roleService,
                                      ShoppingCartService shoppingCartService) {
         this.userService = userService;
         this.shoppingCartService = shoppingCartService;
+        this.roleService = roleService;
     }
 
     @Override
     public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
+        user.setRoles(Set.of(roleService.getByName(Role.RoleName.USER.name())));
```

Comment:
@olekskov , Did you mean in 'RoleService' in 'getByName()' method should be (Role.RoleName roleName) instead of(String roleName)? Or did you mean that it should be there without using 'roleService'?